### PR TITLE
Style/apply clang's complaints @open sesame 7/28 11:45

### PR DESCRIPTION
**Pull request by myungjoo**
Apply clang-format with the .clang-format file of nnstreamer
to all *.cpp files.

Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>

### CODE REVIEW

**Comment by myungjoo** [comment]
In both Ubuntu and Tizen,
if gtest is not included before the inclusion of gstcheck.h, internal-check.h emits errors :(

**Comment by dongju-chae** [comment]
It seems that clang tries to change the format doxygen texts.
How about applying this approach?
- https://stackoverflow.com/questions/48016641/force-clang-format-to-respect-doxygen-block-comments

File: ext/nnstreamer/android_source/gstamcsrc_looper.cc [hunk 1]
```diff
@@ -17,48 +17,50 @@
 /**
  * @file	  gstamcsrc_looper.cc
  * @date	  19 May 2019
- * @brief	  A looper thread to perform event messages between amcsrc and media codec
+ * @brief	  A looper thread to perform event messages between amcsrc and media
+ * codec
```
Comment (dongju-chae) [inline ext/nnstreamer/android_source/gstamcsrc_looper.cc:21]: Q: Did you intend this change? (applying clang format to doxygen comment block)

Comment (myungjoo) [reply ext/nnstreamer/android_source/gstamcsrc_looper.cc:21]: No. That's what clang-format does. We need to adjust clang-format configuration in the future: 1. 80-col limit --> 120-col limit? 2. header include ordering

Comment (dongju-chae) [reply ext/nnstreamer/android_source/gstamcsrc_looper.cc:21]: How about setting `CommentPragmas`?
https://stackoverflow.com/questions/48016641/force-clang-format-to-respect-doxygen-block-comments

Comment (myungjoo) [reply ext/nnstreamer/android_source/gstamcsrc_looper.cc:21]: That didn't help. I gotta find other options.

File: tests/nnstreamer_source/unittest_src_iio.cc [hunk 2]
```diff
@@ -1574,17 +1520,20 @@ TEST (test_tensor_src_iio, data_verify_freq_generic_type)
   clean_iio_dev_structure (dev0);
 }
 
-/**
- * @brief test the unusual/exceptional cases.
- */
-/** @todo Ignore src iio unusual_cases on ubuntu. 
+/** @todo Ignore src iio unusual_cases on ubuntu.
```
Comment (helloahn) [inline tests/nnstreamer_source/unittest_src_iio.cc:1523]: `@brief` has been removed.

File: tests/nnstreamer_sink/unittest_sink.cc [hunk 3]
```diff
@@ -459,559 +475,500 @@ _setup_pipeline (TestOption & option)
...
+        "input=3:160:120:1,3:120:80:1,3:64:48:1 output=3:160:120:1,3:120:80:1,3:64:48:1 inputtype=uint8,uint8,uint8 outputtype=uint8,uint8,uint8 "
+        "inputlayout=NCHW,NHWC,NONE outputlayout=ANY,NHCW,NCHW ! tensor_sink name=test_sink "
+        "videotestsrc num-buffers=%d ! video/x-raw,width=160,height=120,format=RGB,framerate=(fraction)30/1 ! tensor_converter ! mux.sink_0 "
+        "videotestsrc num-buffers=%d ! video/x-raw,width=120,height=80,format=RGB,framerate=(fraction)30/1 ! tensor_converter ! mux.sink_1 "
+        "videotestsrc num-buffers=%d ! video/x-raw,width=64,height=48,format=RGB,framerate=(fraction)30/1 ! tensor_converter ! mux.sink_2",
+        custom_dir ? custom_dir : "./nnstreamer_example", NNSTREAMER_SO_FILE_EXTENSION,
+        option.num_buffers, option.num_buffers, option.num_buffers);
+    break;
+  case TEST_TYPE_CUSTOM_MULTI:
+    /* multiple custom filters */
```
Comment (helloahn) [inline tests/nnstreamer_sink/unittest_sink.cc:767]: Q) `/** COMMENT */` and `/* COMMENT */` what is the right way for the single line comment? there are both of them.

Comment (myungjoo) [reply tests/nnstreamer_sink/unittest_sink.cc:767]: Single line: both are good
Multi line or Doxygen: ```/** */ ```

File: tests/nnstreamer_source/unittest_src_iio.cc (hunk 2 above)
Comment (myungjoo) [reply tests/nnstreamer_source/unittest_src_iio.cc:1523]: They are moved to each function definition line of #if/#else

Comment (helloahn) [reply tests/nnstreamer_source/unittest_src_iio.cc:1523]: got it

**Review by helloahn — APPROVED** [review]
LGTM

**Review by gichan-jang — APPROVED** [review]
LGTM

**Pull request merged**